.travis.yml

```diff
@@ -1,18 +1,48 @@
+sudo: false
+
 language: cpp
 
 compiler:
    - gcc
-   - clang
+#   - clang
 
 os:
    - linux
-   - osx
+#   - osx
```

Comment:
Some of these jobs are temporarily disabled, while I'm testing the coverage.  Will restore all jobs before any PR is approved.